controls/radcolorpicker/styling-and-appearance/appearance.md

```diff
@@ -132,6 +132,21 @@ You can customize the appearance of a __RadColorPicker__ by choosing which of th
 	{{endregion}}
 
 
-##### 4. __RadColorPicker__ is a stylable control. You can customize the look of the RadSplitButton. In order to do this you should use __SplitButtonStyle__ property.
+__4. RadColorPicker is a stylable control. You can customize the look of the RadSplitButton. In order to do this you should use SplitButtonStyle property__
 
-* __SplitButtonStyle__ - this property should be used to control the style of the split button.
\ No newline at end of file
+* __SplitButtonStyle__ - this property should be used to control the style of the split button.
+
+__5. The ColorSelectorStyle gets or sets the color selector style__

```

Comment:
Can you add some information of the RadColorSelector control? Maybe something like: "The ColorSelectorStyle property is used to apply a custom Style to the RadColorSelector control. This is the element shown in the popup opened when you click on the RadColorPicker".